Docs/source/developers/collisions.rst

```diff
@@ -0,0 +1,80 @@
+.. _developers-collisions:
+
+Collisions
+==========
+
+Monte Carlo Collisions
+----------------------
+
+The Monte Carlo collisions (MCC) module can be found in *Source/Particles/Collisions/BackgroundMCC/*.
+Several types of collisions between simulation particles and a neutral background gas are supported including elastic scattering, back scattering, charge exchange, excitation collisions and impact ionization.
+An instance of the class :cpp:class:`MCCProcess` is created for each type of collision included in a simulation. This class saves information about the type of collision and the collision cross-section as a function of energy.
+
+The so-called null collision strategy is used in order to minimize the computational burden of the MCC module. This strategy is standard in PIC-MCC and a detailed description can be found elsewhere, for example in `Birdsall (IEEE Transactions on Plasma Science, vol. 19, no. 2, pp. 65-85, 1991) <https://ieeexplore.ieee.org/document/106800>`_. In short the maximum collision probability is found over a sensible range of energies and is used to pre-select the appropriate number of macroparticles for collision consideration. Only these pre-selected particles are then individually considered for a collision based on their energy and the cross-sections of all the different collisional processes included.
+
+The MCC implementation assumes that the background neutral particles are **thermal**, and are moving at non-relativistic velocities in the lab frame. For each simulation particle considered for a collision, a velocity vector for a neutral particle is randomly chosen. The particle velocity is then boosted to the stationary frame of the neutral through a Galilean transformation. The energy of the collision is calculated using the particle utility function, ``ParticleUtils::getCollisionEnergy()``, as
+
+    .. math:: E_{coll} = \sqrt{(\gamma mc^2 + Mc^2)^2 - (mv)^2} - (mc^2 + Mc^2)
```

Comment:
Doesn't this have to be on a new line to be parsed properly?
```suggestion
    .. math::

       E_{coll} = \sqrt{(\gamma mc^2 + Mc^2)^2 - (mv)^2} - (mc^2 + Mc^2)
```

Reply:
Building the docs locally it rendered fine but maybe there is some difference with different versions of `sphinx`, so I will add the new line to be safe.